internal/glamour/glamour_test.go

```diff
@@ -0,0 +1,137 @@
+package glamour
```

Comment:
It looks like these tests is failing on Windows only, and in the `glamour` repo they are explicitly not running the tests on Windows. Does anyone have an ideas on why these tests are only failing on Windows?